test_sensu.yml

```diff
@@ -4,5 +4,8 @@
   vars:
       firewall_manage_rules: true
   roles:
+    - redis
+    - rabbitmq/sensu
```

Comment:
This also lets us get sensible defaults for things like `sensu_rabbitmq_server`, because we can look up addresses in `hostvars`.